docker-compose.yml

```diff
@@ -1,14 +1,14 @@
-version: '3.0'
 services:
-  leon-ai:
-    container_name: 'leon-ai'
+  leon:
+    container_name: 'leon'
     build:
       context: './'
-    image: 'leonai/leon'
-    ports:
-      - '${LEON_PORT:-1337}:${LEON_PORT:-1337}'
+    image: 'leon-ai/leon'
     environment:
       LEON_PORT: ${LEON_PORT:-1337}
+      LEON_PY_TCP_SERVER_HOST: ${LEON_PY_TCP_SERVER_HOST:-0.0.0.0}
+      LEON_PY_TCP_SERVER_PORT: ${LEON_PY_TCP_SERVER_PORT:-1342}
     stdin_open: true
     tty: true
     command: 'npm run start'
+    network_mode: 'host'
```

Comment:
`network_mode: 'host'` allows us to expose all ports inside the container including `1342` and `1337` (default ports used).